pkg/kubelet/kubelet_volumes_test.go

```diff
@@ -463,6 +463,11 @@ func (f *stubBlockVolume) GetPodDeviceMapPath() (string, string) {
 func (f *stubBlockVolume) SetUpDevice() (string, error) {
 	return "", nil
 }
+
+func (f stubBlockVolume) Map(devicePath, globalMapPath, volumeMapPath, volumeMapName string, podUID types.UID) error {
```

Comment:
To be clear `SetUpDevice(...)` would map to CSI `NodeStageVolume(...)` and the new `Map(...)` would map to `NodePublishVolume(...)` correct?

Reply:
Yes, That was my orig plan.  However, the `GenerateMapVolumeFunc` method used by the controller does several things that would need to be replicated (not a big deal) inside the SetupDevice().